docs/upgrade.html

```diff
@@ -31,6 +31,8 @@
 
 Note: Bumping the protocol version and restarting can be done any time after the brokers were upgraded. It does not have to be immediately after.
 
+Note: Broker IDs above 1000 are now reserved by default to automatically assigned broker IDs. If your cluster has existing broker IDs above that threshold make sure to increase the reserved.broker.max.id broker configuration property accordingly.
```

Comment:
I think we need `<p>` tags around these notes. Currently they just appear on the same line so the second note gets lost.